src/main_WorldViewer.cpp

```diff
@@ -32,11 +32,11 @@ class REGothWorldViewer : public REGoth::REGothEngine
     {
       world = GameWorld::importZEN(WORLD);
 
-      HCharacter hero = world->insertCharacter("PC_HERO", "STARTPOINT");
+      HCharacter hero = world->insertCharacter("PC_HERO", "START_DRAGONISLAND");
```

Comment:
`"START_DRAGONISLAND"` only works for Dragon Island I suppose? :D

Reply:
A method to get the world starting point from the `GameWorld` would be nice to have I guess.

Reply:
Is this something one could easily get from the waynet? Like does it always contain the string `"START"` or something?